src/apprt/gtk/App.zig

```diff
@@ -425,6 +428,7 @@ pub fn init(core_app: *CoreApp, opts: Options) !App {
         // our "activate" call above will open a window.
         .running = c.g_application_get_is_remote(gapp) == 0,
         .css_provider = css_provider,
+        .custom_css_providers = std.ArrayList(*c.GtkCssProvider).init(core_app.alloc),
     };
```

Comment:
If a `std.ArrayListUnmanaged` used then it can have a default value of `.{}` and then doesn't need to be initialized here.